person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person
   attr_accessor :name, :age, :parent_permission
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Please use a getter and setter method for rentals also. In other words, kindly add it to `` attr_accessor``